### PR TITLE
feat: Add Chartist example

### PR DESCRIPTION
**Pull request by Likita**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

**Comment by rpaik** [comment]
@dangreen, can you help with the review?

File: examples/chartist.js/dashboard-app/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "dev": "parcel src/index.html",
+    "without-cube": "parcel src/without-cube/index.html",
+    "build": "parcel build src/index.html"
+  },
+  "devDependencies": {
+    "parcel": "^2.6.2",
+    "process": "^0.11.10"
```
Comment (dangreen) [inline examples/chartist.js/dashboard-app/package.json:12]: Why is this in dependencies?

**Review by dangreen — CHANGES_REQUESTED** [review]

File: examples/chartist.js/dashboard-app/package.json (hunk 1 above)
Comment (Likita) [reply examples/chartist.js/dashboard-app/package.json:12]: @cube-dev/example-wrapper requires it

Comment (igorlukanin) [reply examples/chartist.js/dashboard-app/package.json:12]: Hmm! @ewgenius, does `@cube-dev/example-wrapper` really require any additional deps?

**Pull request closed without merging**